src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/wwwroot/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Real-time Chat with SignalR</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css"
+              crossorigin="anonymous"
+              integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE="/>
+    <link rel="stylesheet" href="/css/site.css" />
+</head>
+<body>
+
+    <header>
+        <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow mb-3">
```

Comment:
You likely have some low-contrast accessibility problems on this page since you use bootstrap, which has that problem by default. I recommend running [Accessibility Insights](https://accessibilityinsights.io/) against your site. The basic automated check (Ctrl-Shift-1) should be enough unless you feel confident in your design sense. Check out [this css](https://github.com/aspnet/AspNetCore/blob/027bd81c25403995200af1a41670d57cddffc57c/src/Identity/UI/src/wwwroot/V4/Identity/css/site.css#L15) for examples at how I fixed contrast problems.

Reply:
@ryanbrandenburg thanks for the feedback. i made your tweaks, used that css you provided, and ran AI on it, which congratulated me for being ok. good suggestion.